example/lib/edge.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:developer';
```

Comment:
This edge.dart file is new for sample app, other files are reviewed in the consent sample app PR.